testing/openimageio/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# Contributor: Mark Riedesel <mark@klowner.com>
+# Maintainer:
+pkgname=openimageio
+pkgver=1.6.14
+pkgrel=0
+pkgdesc="Image i/o library supporting a multitude of image formats"
+url="https://sites.google.com/site/openimageio/"
+arch="all"
+license="BSD"
+depends=""
+makedepends="cmake giflib-dev zlib-dev boost-dev jpeg-dev libpng-dev tiff-dev
+	glew-dev python2-dev openexr-dev ffmpeg2.8-dev libraw-dev libwebp-dev freetype-dev
+	qt-dev mesa-dev openssl-dev jasper-dev opencolorio-dev"
+install=""
+subpackages="$pkgname-python $pkgname-dev $pkgname-doc $pkgname-utils"
+source="$pkgname-$pkgver.tar.gz::https://github.com/OpenImageIO/oiio/archive/Release-${pkgver}.tar.gz
+	nocpp-error.patch
+"
+
+builddir="$srcdir"/oiio-Release-$pkgver
+
+build() {
+	cd "$builddir"
+	mkdir build && cd build
+	cmake .. \
+		-DUSE_OPENSSL=ON \
+		-DUSE_PYTHON=ON \
+		-DUSE_PYTHON3=OFF \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DPYLIB_INSTALL_DIR=lib/python2.7/site-packages \
+		-DOIIO_BUILD_TESTS=OFF \
+		-DOIIO_BUILD_TOOLS=ON
+	make
+}
+
+package() {
+	cd "$builddir"/build
+	make DESTDIR="$pkgdir" install || return 1
+	rm -rf "$pkgdir"/usr/share/fonts/oiio
+}
+
+doc() {
+	default_doc
+}
```

Comment:
`default_doc` is called implicitly when `doc` function is not declared, so you can just omit it.